Sprint-3/revise/implement/get-ordinal-number.js

```diff
@@ -0,0 +1,31 @@
+// In this week's prep, we started implementing getOrdinalNumber
+
+// continue testing and implementing getOrdinalNumber for additional cases
+// Write your tests using Jest - remember to run your tests often for continual feedback
+
+function getOrdinalNumber(number) {
+  const suffixes = ["th", "st", "nd", "rd"];
+  const suffixIndex = number % 10;
+
+  if (suffixIndex === 1 && number !== 11) {
+    return number + "st";
+  } else if (suffixIndex === 2 && number !== 12) {
+    return number + "nd";
+  } else if (suffixIndex === 3 && number !== 13) {
+    return number + "rd";
+  } else {
+    return number + "th";
+  }
+}
```

Comment:
This looks good but can you test for 111, 112 and 113? I don't think your code covers them correctly

Reply:
You were right. I fixed the function now.